docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -48,6 +48,46 @@ If your deployment scripts add explicit NetworkPolicies for {project_name}, you
+
+* It is possible to have both standard and legacy token exchange features enabled, which can be useful if you need to cover standard use cases (internal-internal) together with the other token exchange use cases,
+which are implemented only by legacy token exchange. Like for instance link:{securing_apps_token_exchange_link}#_external-token-to-internal-token-exchange[external to internal token exchange] is implemented only by
+legacy token exchange. In this case {project_name} will serve standard internal-to-internal requests preferably by the standard token exchange when the other requests will be served by the legacy token exchange. It is detected based on the
+parameters if the particular request is standard or not. For instance requests containing non-standard parameters like `requested_issuer` or `requested_subject` are considered legacy.
+
+* Standard token exchange requires to enable switch on the client as described in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-enable[{securing_apps_token_exchange_name}].
+
+Few more clarifications regarding the notable changes in the behaviour of standard token exchange in comparison to legacy token exchange:
+
+* Fine-grained admin permissions are no longer needed and no longer supported for the standard token exchange.
+
+* The most notable change regarding the behaviour of scopes and audiences is, that applied client scopes are based on the client triggering the token exchange request, rather than "target" client specified by the `audience` parameter.
+There is support for multiple values of the `audience` parameter as mentioned in the specification. The details are described in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-scope[{securing_apps_token_exchange_name}].
+
+* Public clients are no longer allowed to send the token exchange requests. Legacy token exchange implemented public clients to exchanging to itself to downscope the original token. This use case can be
+instead covered by using refresh token grant as in the refresh token grant, the parameter `scope` can be actually used to downscope the refreshed access token as mentioned in
+the https://datatracker.ietf.org/doc/html/rfc6749#section-6[OAuth2 specification].
+
+* Exchanging access token to SAML assertion is not supported for now. In other words, it is not supported to use `requested_token_type=urn:ietf:params:oauth:token-type:saml2`.
+
+* Exchanging access token to refresh token is allowed just if explicitly enabled on the client as mentioned in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-details[{securing_apps_token_exchange_name}].
+Right now, it is not supported to request offline tokens or exchange refresh token when the subject token was issued from the offline session. It is recommended to exchange to access tokens instead of
```

Comment:
```suggestion
Currently, it is not supported to request offline tokens or exchange a refresh token when the subject token was issued from an offline session. It is recommended to exchange for access tokens instead of
```